Doc/using/cmdline.rst

```diff
@@ -612,6 +612,26 @@ Miscellaneous options
    .. versionadded:: 3.13
       The ``-X presite`` option.
 
+Controlling Color
+~~~~~~~~~~~~~~~~~
+
+The Python interpreter is configured by default to use colors to highlight
+output in certain situations such as when displaying tracebacks. This
+behavior can be controlled by setting different environment variables.
+
+Setting the environment variable ``TERM`` to ``dumb`` will disable color.
+
+If the environment variable ``FORCE_COLOR`` is set, then color will be
+enabled regardless of the value of TERM. This is useful on CI systems which
+aren’t terminals but can none-the-less display ANSI escape sequences.
+
+If the environment variable ``NO_COLOR`` is set, Python will disable all color
+in the output. This takes precedence over ``FORCE_COLOR``.
+
+All these environment variables are used also by other tools to control color
+output. To control the color output only in the Python interpreter, the
+:envvar:`PY_COLORS` environment variable can be used. This variable takes
+less precedence than ``NO_COLOR`` and ``FORCE_COLOR``.
```

Comment:
In pytest, the precedence seems to be reversed: https://docs.pytest.org/en/7.4.x/reference/reference.html#envvar-PY_COLORS

> `PY_COLORS` takes precedence over `NO_COLOR` and `FORCE_COLOR`

pytest's behaviour makes sense to me, because `PY_COLORS` is more specific (only affects Python) than the others.

Reply:
`PY_COLORS` -> `PYTHON_COLORS` 👍 because `PY_COLORS` is pytest specific.

---

And I think 👍 for `PYTHON_COLORS` taking precedence too.

My thinking is that it's important for accessibility that people should always be able to turn colours off.

So we need to consider situations where other env vars have already been set by others, and maybe you can't change those, but can define new ones. `FORCE_COLOR` and `NO_COLOR` are a little unusual that they're activated when set, regardless of value. So it's important `NO_COLOR` takes precedence over `FORCE_COLOR`.

But I think we'll be fine with `PYTHON_COLORS` taking precedence: if it's already been set to 1=on, I can still set it to 0=off.